src/components/Modals/CreateMeeting/meetingValidations.js

```diff
@@ -42,10 +42,13 @@ const validateBooking = bookingObj => {
   const isValidInvite = validateInvite(bookingObj.attendees);
 
   //Use this for Zulu UTC +00:00
```

Comment:
Keep spaces consistent
```suggestion
  // Use this for Zulu UTC +00:00
```